app/views/posts/index.html.erb

```diff
@@ -1 +1,39 @@
-<h1>user page</h1>
\ No newline at end of file
+<h1>Here is a list of posts for a given user</h1>
+<%= link_to user_path(@user), class: "user-row-link" do %>
+  
+    <div class="user-photo">
+      <%= image_tag @user.photo %>
+    </div>
+  <div class="user-row">
+    <div class="user-info">
+      <h2><%= @user.name %></h2>
+      <div class="post-counter">Number of posts: <%= @user.posts.count %></div>
+    </div>
+  </div>
+<% end %>
+
+<% @user.three_most_recent_posts.each do |post| %>
+  <%= link_to user_post_path(@user, post), class: "post-link" do %>
+    <div class="post-row">
+      <h3><%= post.title %></h3>
+      <p><%= truncate(post.text, length: 50) %></p>
+      <div class="post-stats">
+        <span>Comments: <%= post.comments.count %></span>
+        <span>Likes: <%= post.likes.count %></span>
+      </div>
+    </div>
+  <% end %>
```

Comment:
- [OPTIONAL] I am going to make this optional because it was not mentioned by the previous reviewer. When a user clicks on the `See all Posts` button in the user's show page, it takes the user to the `Post Index Page`. 
According to the requirement, the `Posts index page` is supposed to display all the posts the user has and not the `three_most_recent_posts`. the only place you show the 3 most recent posts is in the user's show page. I'll suggest you remove the `three_most_recent_posts` method in `line 15` and use `@user.posts` to get all the posts belonging to the user. :+1: :smile: :wave:

Reply:
done , thank you @V-Blaze